src/client/views/book/bookEditor.ml

```diff
@@ -44,6 +44,30 @@ let insert t slug i =
 let add t slug =
   insert t slug t.count
 
+let prefill t slug =
+  let%lwt cur = Book.get slug in
+  let%lwt title = Book.title cur in
+  let%lwt date = Book.date cur in
+  let%lwt contents = Book.contents cur in
+  let contents = List.filter_map
+    (function
+           Book.Version _ | Book.InlineSet _ -> None
```

Comment:
Maybe we should be defensive and prevent edition of books containing versions or inline sets? I'm not sure what the best way to handle that is. At the same time, versions/inline sets is very uncommon, beyond "published" books. For the most common case (ball programs), we only have sets, and this feature is helpful already

Reply:
Agreed. Let's keep it at that for now. I'd rather have something imperfect than have nothing at all.